eva_submission/eload_ingestion.py

```diff
@@ -213,17 +257,34 @@ def get_vep_species(self):
         words = self.eload_cfg.query('submission', 'scientific_name').lower().split()
         return '_'.join(words)
 
-    def run_accession_workflow(self):
+    def _generate_csv_mappings_to_ingest(self):
+        vcf_files_to_ingest = os.path.join(self.eload_dir, 'vcf_files_to_ingest.csv')
+        with open(vcf_files_to_ingest, 'w', newline='') as file:
+            writer = csv.writer(file)
+            writer.writerow(['vcf_file', 'assembly_accession', 'fasta', 'report', 'analysis_accession', 'db_name'])
+            analyses = self.eload_cfg.query('brokering', 'analyses')
+            for analysis_alias, analysis_data in analyses.items():
+                assembly_accession = analysis_data['assembly_accession']
+                fasta = analysis_data['assembly_fasta']
+                report = analysis_data['assembly_report']
+                analysis_accession_dict = self.eload_cfg.query('brokering', 'ena', 'ANALYSIS')
+                analysis_accession = list(analysis_accession_dict.keys())[list(analysis_accession_dict.values()).index(analysis_alias)]
```

Comment:
This line alone makes me wonder if we should use the analysis alias as key instead of the accession in the brokering config...